collections/nemo_asr/nemo_asr/parts/jasper.py

```diff
@@ -129,11 +149,11 @@ def __init__(self, inplanes, planes, repeat=3, kernel_size=11, stride=1,
         self.separable = separable
         self.residual_mode = residual_mode
 
-        self.conv = nn.ModuleList()
         inplanes_loop = inplanes
+        conv = nn.ModuleList()
 
-        if tied:
```

Comment:
I see that we are removing the weight sharing option. Can we then remove it from the class initializer as a parameter?

Reply:
done